### PR TITLE
Chore/315 remove blueprint from token dapp

### PR DESCRIPTION
**Pull request by mattrussell36**
# Related issues 🔗

Partially addresses #315 

# Description ℹ️

Removes @blueprintjs/core and @blueprintjs/select dependencies from the token dapp, adding components where necessary to replace functionality

# Demo 📺

New radio buttons:
Screen Shot 2022-05-18 at 8 36 06 AM

# Technical 👨‍🔧

- Mobile menu drawer has been replaced with a custom dialog component
- Country select used to use the blueprints suggest component but I've opted to just replace this with a standard Select component from the ui-toolkit for now. This means we lose the type to search for country functionality.
- All imported blueprint styles now don't need to be bundled
- Fixes spacing on disassociate page

### CODE REVIEW

File: apps/token/src/components/connected-vega-key/connected-vega-key.tsx.tsx [hunk 1]
```diff
@@ -9,7 +9,9 @@ export const ConnectedVegaKey = ({ pubKey }: { pubKey: string | null }) => {
       <strong data-testid="connected-vega-key-label">
```
Comment (sam-keen) [inline apps/token/src/components/connected-vega-key/connected-vega-key.tsx.tsx:9]: Question (non-blocking): what's the deal with this file name - typo?

File: apps/token/src/components/country-selector/country-selector.tsx [hunk 2]
```diff
@@ -36,30 +13,18 @@ export const CountrySelector = ({
...
-        popoverProps={{ minimal: true }}
-        noResults={<MenuItem disabled={true} text="No results." />}
-        itemPredicate={filterCountry}
-        fill={true}
-      />
+      <Select
+        value={code ? code : undefined}
+        onChange={(e) => onSelectCountry(e.target.value)}
+      >
+        {countryData.map((country) => {
```
Comment (sam-keen) [inline apps/token/src/components/country-selector/country-selector.tsx:20]: Chore (non-blocking): implicit return and destructuring cleaner here IMO

File: apps/token/src/components/transactions-modal/transactions-modal.tsx [hunk 3]
```diff
@@ -65,53 +48,52 @@ export const TransactionModal = () => {
...
+        <table className="w-full">
+          <thead>
+            <tr>
+              <TransactionModalTh>{t('transaction')}</TransactionModalTh>
+              <TransactionModalTh>{t('status')}</TransactionModalTh>
+            </tr>
+          </thead>
+          <tbody>
+            {transactions.map((t) => {
+              return (
```
Comment (sam-keen) [inline apps/token/src/components/transactions-modal/transactions-modal.tsx:80]: Suggestion (non-blocking): could do implicit return here too (clean, reduces lines of code, easy to read, win win win IMO)

File: apps/token/src/routes/staking/staking-form.tsx [hunk 4]
```diff
@@ -210,28 +209,26 @@ export const StakingForm = ({
       <h2 className="text-h4 mb-8">{t('Manage your stake')}</h2>
       <FormGroup>
         <RadioGroup
-          onChange={(e) => {
+          onChange={(value) => {
```
Comment (sam-keen) [inline apps/token/src/routes/staking/staking-form.tsx:212]: Question (non-blocking): Is the ts-ignore still necessary on 213 now?

**Review by sam-keen — APPROVED** [review]
A few non-blocking bits from me, looks good.

File: libs/tailwindcss-config/src/theme.js [hunk 5]
```diff
@@ -126,7 +126,7 @@ module.exports = {
     DEFAULT: '0.225rem',
     md: '0.3rem',
     lg: '0.5rem',
-    full: '9999px',
+    full: '100%',
```
[the remark this answers is not in the document]
Comment (mattrussell36) [reply libs/tailwindcss-config/src/theme.js:129]: Pretty sure that's not the case for border radius and it goes by the actual element. @bglownia 9999px seemed like weird arbitrary value for `rounded-full` any reason you use it over 100%?

[the remark this answers is not in the document]
Comment (sam-keen) [reply libs/tailwindcss-config/src/theme.js:129]: Ah scratch my comment, I thought you were adjusting breakpoints (just saw the md, lg etc). Value of 100% seems like a good move to me.

**Pull request merged**